python/mxnet/library.py

```diff
@@ -24,6 +24,17 @@
 from .ndarray.register import _make_ndarray_function
 from .symbol.register import _make_symbol_function
 
+class MXlib:
+    """Holds a pointed to a loaded shared library and closes it on destruction"""
+    def __init__(self, handle):
+        self.handle = handle
+    def __del__(self):
+        libdl = ctypes.CDLL("libdl.so")
+        libdl.dlclose(self.handle)
```

Comment:
so it is to elaborate libdl.so lives longer than libmxnet.so?

Reply:
this is how we close the library we loaded before, by using `libdl.so` to call `dlcose`